.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = csrc/velox/velox
 	url = https://github.com/facebookincubator/velox.git
 
+[submodule "csrc/velox/re2"]
+	path = csrc/velox/re2
+	url = https://github.com/google/re2.git
```

Comment:
Velox requires re2. So as long as TorchArrow depends on re2, you do not need to add this submodule. re2 target is already defined. (also I do not see you add something like `add_subdirectory(re2)`

https://github.com/facebookincubator/velox/blob/e4942d0ae32585c48a0faa8f344cdb01bd02ed71/velox/functions/lib/CMakeLists.txt#L14